### ISSUE TITLE
List of allowed values not displayed on model for collection of enum

### ISSUE THREAD

**Issue by ddowling-act** [swagger-api/swagger-ui#1381]
When the following is present in a swagger.json:

```
    "aCollectionOfEnums": {
      "type": "array",
      "readOnly": true,
      "uniqueItems": true,
      "items": {
        "type": "string",
        "enum": [
          "ENTRY1",
          "ENTRY2",
          "ENTRY3",
          "ENTRY4",
          "ENTRY5",
          "ENTRY6"
        ]
      }
    }
```

the model only displays "aCollectionOfEnums(Array[string])", with the model schema showing   

```
"aCollectionOfEnums": [
"ENTRY1"
]
```

With enums not in a collection, the model will show all acceptable values, but not when they are within a collection.

**Comment by webron** [comment]
Which version of the UI do you try it with?

**Comment by senasi** [comment]
I can confirm this using UI version 2.1.0. When you move `enum` definition one level up, it will display model correctly.

Using example above, moving `enum` one level up:
```
 "aCollectionOfEnums": {
      "type": "array",
      "readOnly": true,
      "uniqueItems": true,
      "items": {
        "type": "string"
      },
      "enum": [
          "ENTRY1",
          "ENTRY2",
          "ENTRY3",
          "ENTRY4",
          "ENTRY5",
          "ENTRY6"
        ]
    }
```
displays:
![enum](https://cloud.githubusercontent.com/assets/12956618/8267328/963c9a60-175c-11e5-829c-151c989e28e9.png)
but model schema will change to:
```
{
  "aCollectionOfEnums": [
    "string"
  ]
}
```

**Comment by ddowling-act** [comment]
This was with the latest version from master as of Friday

**Comment by kulkarniankita** [comment]
Has this been fixed or is anyone fixing it?

**Comment by cbornet** [comment]
:+1

**Comment by fubar** [comment]
Using a definition for the enum works as well:
```json
"definitions": {
  "SocialNetwork": {
    "type": "string",
    "enum": [
      "TWITTER",
      "FACEBOOK"
    ]
  }
}
```
Then:
```json
"Foo": {
  "type": "object",
  "properties": {
    "socialNetworks": {
      "type": "array",
      "items": {
        "$ref": "#/definitions/SocialNetwork"
      }
    }
  }
}
```

**Comment by webron** [comment]
This should be resolved in 3.0. Please reopen if not.
